### ISSUE TITLE
Add a "build" objective type for Gunsmith style quests

### ISSUE THREAD

**Issue by thaddeus** [TarkovTracker/tarkovdata#2]
Need to add build objectives for quests like Gunsmith. Probably should just have a target of the type of weapon you'll need to build, maybe could include require stats or items as a 'with' array.

**Comment by Monksman45** [comment]
What do you think of the following format? 
```
   {
        "id": 82,
        "require": {
            "level": 10,
            "quests": []
        },
        "giver": "Mechanic",
        "turnin": "Mechanic",
        "title": "Gunsmith - Part 1",
        "wiki": "https://escapefromtarkov.gamepedia.com/Gunsmith_-_Part_1",
        "exp": 6500,
        "unlocks": [],
        "reputation": [
            {
                "trader": "Mechanic",
                "rep": 0.04
            }
        ],
        "objectives": [
            {
                "type": "build",
                "target": "MP-133",
                "number": 1,
                "with": [
                    {
                        "type":"cells",
                        "value":"<4"
                    },
                    {
                        "type":"stat",
                        "name":"ergonomics",
                        "value":">47"
                    },
                    {
                        "type":"stat",
                        "name":"recoil",
                        "value":"<950"
                    },
                    {
                        "type":"attachment",
                        "name":"Laser Designator"
                    },
                    {
                        "type":"attachment",
                        "name":"Extended Mag"
                    }
                ],
                "id": 162
            }
        ]
    }
```

**Comment by thaddeus** [comment]
Yeah - that looks like it would cover all the important information - solid!

**Comment by thaddeus** [comment]
Resolved by #128
